### ISSUE TITLE
Install script is broken.

### ISSUE THREAD

**Issue by tlstommy** [ryanwa18/spotipi-eink#2]
The install script is not saving the client token properly.

**Comment by heytcass** [comment]
Agreed. It no longer directs you to a URL to copy into your browser. Also .cache seems to not be saved at all.

**Comment by Gabbajoe** [comment]
for me the installation script was working fine. including the redirection. anyway as i bought a smaller version of the screen and loved to used the buttons on the screen for actions i forked this project and did some code changes including rework of the setup.sh see: https://github.com/Gabbajoe/spotipi-eink maybe this works for you both.

**Comment by joasmund** [comment]
For me it does not find the module spotipy when installing, so i never get the link I am supposed to use.

**Comment by Gabbajoe** [comment]
> @Gabbajoe if I wanted to display this app on the following LCD screen --> [Here](https://www.waveshare.com/wiki/3.2inch_RPi_LCD_(B)), would that be easy for me to do with little experience? I tried to poke around the source code, but not sure how to modify it.

Hi @Peternbrown this might not the best way to ask questions in a issue of a totally different topic :) but anyhow. The display that you linked is acting as a regular display, with major changes this is not possible..

**Comment by conrat4567** [comment]
> for me the installation script was working fine. including the redirection. anyway as i bought a smaller version of the screen and loved to used the buttons on the screen for actions i forked this project and did some code changes including rework of the setup.sh see: https://github.com/Gabbajoe/spotipi-eink maybe this works for you both.

Hey, just seen this, your script is still working but my Pi gets stuck on Preparing metadata when running the spotipi script. Any ideas?

**Comment by Gabbajoe** [comment]
> Hey, just seen this, your script is still working but my Pi gets stuck on Preparing metadata when running the spotipi script. Any ideas?

if you use a raspberry pi zero (1) the pillow installation takes ages. ~ 15 min or so. I updated the setup.sh and the requirements recently in my repo to reflect the latest raspberry p os changes.
